app/helpers/sessions_helper.rb

```diff
@@ -1,11 +1,2 @@
 module SessionsHelper
-
-  def logged_in?
-    !current_user.nil?
-  end
-
-  def current_user
-    @current_user ||= User.find_by(id: session[:user_id])
-  end
-
-end
\ No newline at end of file
+end
```

Comment:
Might delete this file since it's empty and you've moved all of this to `application_controller`